draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,79 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Attester {#compositeattester}
+
+A Composite Attester is an entity composed of multiple sub-entities that its
+trustworthiness has to be determined by evaluating all these sub-entities.
+Each sub-entity has at least one Attesting Environment collecting the claims from
+at least one Target Environment and generates its Evidence against its trustworthiness,
+therefore it can be called an Attester. Among these Attesters, there may be only some,
+which can be called Main Attesters, that have the communication functionality
+with the Verifier, and other Attesters are evaluated via their help.
+
+For example, the carrier-grade router is a composite device consisted of multiple slots.
+The trustworthiness of the router depends on all its slots' trustworthiness.
```

Comment:
Is it worth distinguishing between sub-entities that are isolated (ethernet-like connectivity only), and those that might have direct memory access (PCIe, etc.)?  This matters in the case of composite BMC capability.

Reply:
In my examples, all the sub-entities (the slots of the router and the routers of the multi-chassis router) are isolated and they only have the ethernet-like connectivity between each other. Maybe there are other examples in which sub-entities have direct memory access, but I don't know what exact examples are. Besides, I don't figure out what composite BMC capability is and how it is affected.